trestle/common/common_types.py

```diff
@@ -39,6 +37,4 @@
 )
 
 OBT = TypeVar('OBT', bound=OscalBaseModel)
-
-# model types containing uuids that should not regenerate
-FixedUuidModel = common.Resource
```

Comment:
FiixedUuidModel is meant to be an expandable list of oscal types with uuids that should not regenerate.  So please leave this logic in.  There may be others that should not have uuid changed.